s3-backed-core/src/main/java/com/bakdata/kafka/AbstractS3BackedConfig.java

```diff
@@ -73,6 +75,16 @@
     public static final String S3_SECRET_KEY_CONFIG = PREFIX + "secret.key";
     public static final String S3_SECRET_KEY_DOC =
             "AWS secret key to use for connecting to S3. Leave empty if AWS credential provider chain should be used.";
```

Comment:
Please also update the doc here. It is always the same doc as in the README. Also Update the STS doc accordingly please. The README one is very good!

Reply:
Sure thing. I actually copied these ones from the Kafka Connect configs, but I modified them in the README to make more sense in the context of the other configs.

Reply:
@philipp94831 done

Reply:
@Tomczik76 I see that you update the STS doc but please also update the doc for secret_key and access_key as you did in the README :)